src/ripple/app/ledger/Ledger.cpp

```diff
@@ -680,10 +680,14 @@ loadLedgerHelper(std::string const& sqlSuffix)
         rangeCheckedCast<std::uint32_t>(ledgerSeq64.value_or (0));
 
     uint256 prevHash, accountHash, transHash;
-    ledgerHash.SetHexExact (sLedgerHash.value_or(""));
-    prevHash.SetHexExact (sPrevHash.value_or(""));
-    accountHash.SetHexExact (sAccountHash.value_or(""));
-    transHash.SetHexExact (sTransHash.value_or(""));
+    if (sLedgerHash)
```

Comment:
A free and concise way to address Nik's comment is:
````
uint256 prevHash{};
uint256 accountHash{};
// ...
````
This gives us leeway in the future to make `uint256` behave like `std::chrono::duration`:  It doesn't initialize for default-initialization and it zero-initializes when given braces (or parentheses) as above.